backend/src/models/studentProfile.js

```diff
@@ -8,6 +8,14 @@ export class StudentProfile {
 
   async getStudentsByGuardian(id) {
     const client = await con.connect();
+    
+    client.query("SELECT * FROM tutors", (error, results) => {
```

Comment:
Could you please update the sql to join on users to retrieve everything at the same time, sample statement below
`select t.*, u.firstname, u.lastname from tutors t join users u on u.user_id = t.tutor_id;`